packages/remix/src/client/uiComponents.tsx

```diff
@@ -6,12 +6,14 @@ import { useClerkRemixOptions } from './RemixOptionsContext';
 
 export const SignIn = (props: SignInProps) => {
   const { signInUrl } = useClerkRemixOptions();
+  const { routing: _routing, path: _path, ...restProps } = props;
+
   if (signInUrl) {
     return (
       <BaseSignIn
         routing='path'
         path={signInUrl}
-        {...props}
+        {...restProps}
```

Comment:
same but for SignIn : https://github.com/clerk/javascript/pull/1957/files#r1389065093

Reply:
Fixed, you can check my comment here https://github.com/clerk/javascript/pull/1957#discussion_r1389688018